containers/Investment/RevolvingPool/View/PoolOverview.tsx

```diff
@@ -1,53 +1,192 @@
 import * as React from 'react'
 import { Box, Heading, Table, TableBody, TableRow, TableCell } from 'grommet'
-import { Pool } from '../../../../config'
+import { connect, useSelector } from 'react-redux'
+import { loadPool, PoolDataV3, PoolState } from '../../../../ducks/pool'
+import { toPrecision } from '../../../../utils/toPrecision'
+import { addThousandsSeparators } from '../../../../utils/addThousandsSeparators'
+import { baseToDisplay, feeToInterestRate } from '@centrifuge/tinlake-js'
+import { TINRatioBar } from '../../../../components/TINRatioBar/index'
+import {
+  TokenLogo,
+  BalanceSheetDiagram,
+  BalanceSheetDiagramLeft,
+  BalanceSheetMidLine,
+  BalanceSheetDiagramRight,
+  DividerTop,
+  DividerBottom,
+  DividerInner,
+} from './styles'
+import BN from 'bn.js'
 
-interface Props {
-  pool: Pool
-}
+interface Props {}
 
 const PoolOverview: React.FC<Props> = () => {
+  const pool = useSelector<any, PoolState>((state) => state.pool)
+
+  const poolValue =
+    (pool.data !== null &&
+      (pool.data as PoolDataV3).netAssetValue &&
+      (pool.data as PoolDataV3).reserve &&
+      (pool.data as PoolDataV3).netAssetValue.add((pool.data as PoolDataV3).reserve)) ||
+    '0'
+  const dropRate = (pool.data && pool.data.senior && pool.data.senior.interestRate) || '0'
```

Comment:
Can we start using this syntax everywhere for brevity? 

```suggestion
  const dropRate = (pool.data?.senior?.interestRate) || '0'
```